src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -794,6 +794,10 @@ public AddAuthDomainResponseModel addSnapshotDataAccessControls(
         .submitAndWait(AddAuthDomainResponseModel.class);
   }
 
+  public List<String> getAuthDomains(UUID snapshotId, AuthenticatedUserRequest userReq) {
```

Comment:
Should this be `getAuthDomain`? Or the iamService method be `retrieveAuthDomains`? Its confusing to me that one is plural and one is singular.